### DISCUSSION TITLE
Lidemy 討論區新手指南

### DISCUSSION THREAD

**Discussion by aszx87410** [Lidemy/forum#26] [General]
## 👋  哈囉!

這是 GitHub 正在 beta 的新功能，叫做 Discussions，簡單來說就是個討論區  
沒意外的話這邊會變成未來 Lidemy 這個平台上各種問題的討論區

目前已經有建好幾個分類讓大家提問，包括：

1. 前端相關問題：有關前端的都來這邊問，像是 HTML, CSS, JS, React...
2. 後端相關問題：例如說 PHP, Express 等等
3. 其他問題：像是 Command line, Git, NET101 相關的問題都可以來這邊
4. 程式導師實驗計畫：程式導師實驗計畫每週作業相關問題

在標題的部分要請大家符合一下規範，在標題前面加上要問的問題的相關課程的課號  
舉例來說，有一堂課叫做 `[JS101] 用JavaScript 一步步打造程式基礎`  
那你的標題就應該是：`[JS101] 想問一下 map 的使用方法`
這樣比較方便之後的人搜尋相關的內容，就可以用課號來找問題

在提問前請先利用搜尋功能，看看有沒有人問過類似的問題
如果是程式導師實驗計畫的就是 `[MTR04] 想問一下 week6 作業` 這樣子

**Comment by Wozski** [comment]
我覺得有點 Cool！

**Reply by YongChenSu** [reply]
超酷

**Comment by Rober501** [comment]
[js101] 請問 老師教學影片中 用來寫 Javascript 程式的介面使甚麼? 我用　vim　練習　覺得很不方便. 可以用visual studio 來練習寫嗎？　需要設定連結　node.js　嗎?  老師用的教學介面 看起來很方便 如何安裝呢?  謝謝!

**Reply by aszx87410** [reply]
可以用 sublime 或是 vscode 喔

**Comment by powerfultraveling** [comment]


**Reply by aszx87410** [reply]
不是在這邊問喔，是要另外發一篇，然後要附上 devtool 那個 req 的截圖，看那邊最準

**Reply by chachachater** [reply]
是要去這邊問喔，https://github.com/Lidemy/forum/discussions
我之前也遇到這個問題，那時候發現原因是因為$_POST 只能取得 Content-type 為 application/x-www-form-urlencoded 或 multipart/form-data 的資料，而我的 AJAX (jQuery)沒有指定 content type，猜你可能是這個原因造成的。[資料](https://xyz.cinc.biz/2013/06/php-filegetcontents-post.html)

> PHP文件: An associative array of variables passed to the current script via the HTTP POST method when using application/x-www-form-urlencoded or multipart/form-data as the HTTP Content-Type in the request.
